esignet-service/scripts/oauth-smoke.sh

```
@@ -2,6 +2,16 @@
 set -uo pipefail
 
 BASE_URL="${BASE_URL:-http://127.0.0.1:8080}"
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+REDIRECT_URI="https://localhost:3000"
+SCOPE="openid profile email"
```

Shellcheck (0.11.0)

[warning] SCOPE appears unused. Verify use (or export if used externally).

(SC2034)